kaldb/src/main/proto/metadata.proto

```diff
@@ -147,3 +147,29 @@ message DatasetPartitionMetadata {
   // List of Kafka partition ids
   repeated string partitions = 3;
 }
+
+// Schema definitions.
+message LuceneFieldDef {
+  // Name of the field
+  string name = 1;
+
+  // Type of the field
+  string type = 2;
+
+  // Store the value as a Lucene StoredField.
+  bool isStored = 3;
```

Comment:
These fields aren't using the correct naming convention, which will fail linting - see https://github.com/slackhq/kaldb/pull/222